src/config/dynamic-theme-fixes.config

```diff
@@ -3070,6 +3070,9 @@ CSS
 .generator_color.is-light {
     color: #000 !important;
 }
+.color-picker_saturation {
+    filter: invert(100%) brightness(110%) !important;
```

Comment:
```suggestion
    background-image: linear-gradient(right,rgba(255,255,255,0),#fff) !important;
```

Reply:
That didn't work for me, I think that while it's not the way to do it, it still works at least I guess?